addons/account_analytic_default/models/account_analytic_default.py

```diff
@@ -24,8 +24,8 @@ class AccountAnalyticDefault(models.Model):
 
     @api.constrains('analytic_id', 'analytic_tag_ids')
     def _check_account_or_tags(self):
-        if any(not default.analytic_id and not default.analytic_tag_ids for default in self):
-            raise ValidationError(_('An analytic default requires at least an analytic account or an analytic tag.'))
+        if any(not default.analytic_id and not default.analytic_tag_ids or any(not tag.analytic_distribution_ids for tag in default.analytic_tag_ids) for default in self):
```

Comment:
```suggestion
        if any(not default.analytic_id and not default.analytic_tag_ids or not any(tag.analytic_distribution_ids for tag in default.analytic_tag_ids) for default in self):
```

Doesn't this make more sense?

Also, please indent/spread on multiple lines

Reply:
> not the clearest comment sorry.

keep this